tools/install.sh

```diff
@@ -43,19 +62,25 @@ mv -f ~/.zshrc-omztemp ~/.zshrc
 
 TEST_CURRENT_SHELL=$(expr "$SHELL" : '.*/\(.*\)')
 if [ "$TEST_CURRENT_SHELL" != "zsh" ]; then
-    echo "\033[0;34mTime to change your default shell to zsh!\033[0m"
+    printf "${BLUE}Time to change your default shell to zsh!${NORMAL}\n"
     chsh -s $(grep /zsh$ /etc/shells | tail -1)
 fi
 unset TEST_CURRENT_SHELL
 
-echo "\033[0;32m"'         __                                     __   '"\033[0m"
-echo "\033[0;32m"'  ____  / /_     ____ ___  __  __   ____  _____/ /_  '"\033[0m"
-echo "\033[0;32m"' / __ \/ __ \   / __ `__ \/ / / /  /_  / / ___/ __ \ '"\033[0m"
-echo "\033[0;32m"'/ /_/ / / / /  / / / / / / /_/ /    / /_(__  ) / / / '"\033[0m"
-echo "\033[0;32m"'\____/_/ /_/  /_/ /_/ /_/\__, /    /___/____/_/ /_/  '"\033[0m"
-echo "\033[0;32m"'                        /____/                       ....is now installed!'"\033[0m"
-echo "\n\n \033[0;32mPlease look over the ~/.zshrc file to select plugins, themes, and options.\033[0m"
-echo "\n\n \033[0;32mp.s. Follow us at http://twitter.com/ohmyzsh.\033[0m"
-echo "\n\n \033[0;32mp.p.s. Get stickers and t-shirts at http://shop.planetargon.com.\033[0m"
+printf "${GREEN}"
+echo '         __                                     __   '
+echo '  ____  / /_     ____ ___  __  __   ____  _____/ /_  '
+echo ' / __ \/ __ \   / __ `__ \/ / / /  /_  / / ___/ __ \ '
+echo '/ /_/ / / / /  / / / / / / /_/ /    / /_(__  ) / / / '
+echo '\____/_/ /_/  /_/ /_/ /_/\__, /    /___/____/_/ /_/  '
+echo '                        /____/                       ....is now installed!'
+echo ''
+echo ''
+echo 'Please look over the ~/.zshrc file to select plugins, themes, and options.'
+echo ''
+echo 'p.s. Follow us at https://twitter.com/ohmyzsh.'
+echo ''
+echo 'p.p.s. Get stickers and t-shirts at http://shop.planetargon.com.'
+echo ''
+printf "${NORMAL}"
 env zsh
```

Comment:
One other data point: #4261 is a user who was confused about OMZ's normal startup sequence by the "preview" subshell in the installer, and would also like an option to avoid it, to support scripted installs.

Is there an existing issue for discussion of what the installer behavior should be here?